GUI/UI/UIPartyCreation.cpp

```diff
@@ -757,7 +748,7 @@ GUIWindow_PartyCreation::GUIWindow_PartyCreation() :
         CreateButton(uX, 120, 145, 25, 1, 0, UIMSG_PlayerCreationChangeName, uControlParam);
         uX += 158;
         ++uControlParam;
-    } while ((signed int)uX < window->GetWidth());
+    } while ((signed int)uX < 640);  // 640
```

Comment:
could you please user renwidth here as well?

Reply:
changed to for loop